proxy.cabal

```diff
@@ -11,20 +11,28 @@ author: Amir Sabbaghi
 data-dir: ""
 
 executable proxy
-    build-depends: base == 4.8.*, network >=2.6 && <2.7, proxy
+    build-depends: base < 4.8, network >=2.6 && <2.7, proxy
```

Comment:
`base 4.8` is for GHC `7.10.1`, but we're using `7.8.4`.

https://wiki.haskell.org/Base_package